lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -25,27 +25,29 @@
 ---
 module: nxos_install_os
 extends_documentation_fragment: nxos
-short_description: Set boot options like boot image and kickstart image.
+short_description: Set boot options like boot, kickstart image and issu.
 description:
     - Install an operating system by setting the boot options like boot
-      image and kickstart image.
+      image and kickstart image and optionally select to install using
+      ISSU (In Server Software Upgrade).
 notes:
-    - Tested against NXOSv 7.3.(0)D1(1) on VIRL
-    - The module will fail due to timeout issues, but the install will go on
-      anyway. Ansible's block and rescue can be leveraged to handle this kind
-      of failure and check actual module results. See EXAMPLE for more about
-      this. The first task on the rescue block is needed to make sure the
-      device has completed all checks and it started to reboot. The second
-      task is needed to wait for the device to come back up. The last two tasks
-      are used to verify the installation process was successful.
+    - Tested against the following platforms and images
```

Comment:
From offline review comments:
* I need to add additional versions for platforms validated in this documentation.

Reply:
Complete.